Invoking command in '/home/ros2_ws/src/pradobot_ultrasonic': PYTHONPATH=/home/ros2_ws/src/build/pradobot_ultrasonic/prefix_override:/home/ros2_ws/src/install/pradobot_ultrasonic/lib/python3.8/site-packages:${PYTHONPATH} /usr/bin/python3 setup.py egg_info --egg-base ../build/pradobot_ultrasonic build --build-base /home/ros2_ws/src/build/pradobot_ultrasonic/build install --record /home/ros2_ws/src/build/pradobot_ultrasonic/install.log --single-version-externally-managed
Invoked command in '/home/ros2_ws/src/pradobot_ultrasonic' returned '0': PYTHONPATH=/home/ros2_ws/src/build/pradobot_ultrasonic/prefix_override:/home/ros2_ws/src/install/pradobot_ultrasonic/lib/python3.8/site-packages:${PYTHONPATH} /usr/bin/python3 setup.py egg_info --egg-base ../build/pradobot_ultrasonic build --build-base /home/ros2_ws/src/build/pradobot_ultrasonic/build install --record /home/ros2_ws/src/build/pradobot_ultrasonic/install.log --single-version-externally-managed
